client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -219,6 +231,7 @@ export default compose( [
 				showClone: 'active' === rewindState.state && ! isAtomic,
 				showDeleteContent: isAtomic || ( ! isJetpack && ! isVip && ! isP2Hub ),
 				showDeleteSite: ( ! isJetpack || isAtomic ) && ! isVip && sitePurchasesLoaded,
+				showHeaderCode: isJetpack || isAtomic,
```

Comment:
Seems like this hints that WP.com simple sites won't be supported. We should make sure to handle these site types properly at the route level too (see my comment above)

Reply:
See comment below